src/transformers/models/pegasus/tokenization_pegasus_fast.py

```diff
@@ -106,6 +106,7 @@ def __init__(
         unk_token="<unk>",
         mask_token="<mask_2>",
         mask_token_sent="<mask_1>",
+        remove_space=True,
```

Comment:
Is this non-breaking?

Reply:
Actually this was unnecessary so I removed it.